content/en/docs/concepts/services-networking/service.md

```diff
@@ -827,434 +628,268 @@ metadata:
 ```
 
 {{% /tab %}}
-{{% tab name="IBM Cloud" %}}
+{{% tab name="Baidu Cloud" %}}
 
 ```yaml
 [...]
 metadata:
     name: my-service
     annotations:
-        service.kubernetes.io/ibm-load-balancer-cloud-provider-ip-type: "private"
+        service.beta.kubernetes.io/cce-load-balancer-internal-vpc: "true"
 [...]
 ```
 
 {{% /tab %}}
-{{% tab name="OpenStack" %}}
+{{% tab name="GCP" %}}
 
 ```yaml
 [...]
 metadata:
     name: my-service
     annotations:
-        service.beta.kubernetes.io/openstack-internal-load-balancer: "true"
+        cloud.google.com/load-balancer-type: "Internal"
 [...]
 ```
 
 {{% /tab %}}
-{{% tab name="Baidu Cloud" %}}
+{{% tab name="IBM Cloud" %}}
 
 ```yaml
 [...]
 metadata:
     name: my-service
     annotations:
-        service.beta.kubernetes.io/cce-load-balancer-internal-vpc: "true"
+        service.kubernetes.io/ibm-load-balancer-cloud-provider-ip-type: "private"
 [...]
 ```
 
 {{% /tab %}}
-{{% tab name="Tencent Cloud" %}}
+{{% tab name="OCI" %}}
 
 ```yaml
 [...]
 metadata:
-  annotations:
-    service.kubernetes.io/qcloud-loadbalancer-internal-subnetid: subnet-xxxxx
+    name: my-service
+    annotations:
+        service.beta.kubernetes.io/oci-load-balancer-internal: true
 [...]
 ```
+ {{% /tab %}}
 
-{{% /tab %}}
-{{% tab name="Alibaba Cloud" %}}
+{{% tab name="OpenStack" %}}
 
 ```yaml
 [...]
 metadata:
-  annotations:
-    service.beta.kubernetes.io/alibaba-cloud-loadbalancer-address-type: "intranet"
+    name: my-service
+    annotations:
+        service.beta.kubernetes.io/openstack-internal-load-balancer: "true"
 [...]
 ```
 
 {{% /tab %}}
-{{% tab name="OCI" %}}
+{{% tab name="Tencent Cloud" %}}
 
 ```yaml
 [...]
 metadata:
-    name: my-service
-    annotations:
-        service.beta.kubernetes.io/oci-load-balancer-internal: true
+  annotations:
+    service.kubernetes.io/qcloud-loadbalancer-internal-subnetid: subnet-xxxxx
 [...]
 ```
+
 {{% /tab %}}
 {{< /tabs >}}
 
-#### TLS support on AWS {#ssl-support-on-aws}
+### `type: ExternalName` {#externalname}
 
-For partial TLS / SSL support on clusters running on AWS, you can add three
-annotations to a `LoadBalancer` service:
+{{< note >}}
+You need either `kube-dns` version 1.7 or CoreDNS version 0.0.8 or higher
+to use the `ExternalName` type.
+{{< /note >}}
```

Comment:
I think we can omit this nowadays, externalname is part of the DNS specification https://github.com/kubernetes/dns/blob/master/docs/specification.md#25---records-for-external-name-services , any Kubernetes DNS implementation has to support them to be conformant

Reply:
If you definitely want to see that change made @aojea, would you be willing to make a PR against _main_ to omit that text?

See https://kubernetes.io/docs/concepts/services-networking/service/#publishing-services-service-types for the current version.